app/aws-lsp-json-runtimes/src/tests/jsonServerCFInteg.test.ts

```diff
@@ -12,41 +12,51 @@ import {
     TEXT_TO_DIAGNOSE_JSON,
     TEXT_TO_FORMAT_JSON,
     TEXT_TO_HOVER_JSON,
+    DIAGNOSTICS_RESPONSE_JSON_CUSTOM,
+    HOVER_JSON_CUSTOMIZED,
 } from './testUtilsCF'
 
-describe('Test JsonServer with CloudFormation schema', () => {
+async function createLSPServer(runtimeFile: string) {
     const rootPath = path.resolve(__dirname)
     let process: ChildProcessWithoutNullStreams
-    let endpoint: JSONRPCEndpoint
-    let client: LspClient
-    const runtimeFile = path.join(__dirname, '../../', 'build', 'aws-lsp-json-standalone.js')
 
-    before(async () => {
-        // start the LSP server
-        process = spawn('node', [runtimeFile, '--stdio'], {
-            shell: true,
-            stdio: 'pipe',
-        })
+    // Start the LSP server
+    // eslint-disable-next-line prefer-const
+    process = spawn('node', [runtimeFile, '--stdio'], {
+        shell: true,
+        stdio: 'pipe',
+    })
```

Comment:
With your changes you can also change the `process` assignment to a `const` assignment and that would result in not having to disable the `eslint` rule

```suggestion
    // Start the LSP server
    const process = spawn('node', [runtimeFile, '--stdio'], {
        shell: true,
        stdio: 'pipe',
    })
```